AdminUi/test/AdminUi.Tests.Integration/Models/HttpResponse.cs

```diff
@@ -9,6 +10,8 @@ public class HttpResponse<T>
     public bool IsSuccessStatusCode { get; set; }
     public string? ContentType { get; set; }
     public string? RawContent { get; set; }
+    public IReadOnlyCollection<HeaderParameter>? Headers { get; set; }
+    public CookieCollection? Cookies { get; set; }
```

Comment:
Don't use RestSharp classes here. We do not want to expose these to the rest of the tests, so that we are able to switch the used http client without having to touch all the tests.

Reply:
Why did you even add them? They aren't used anywhere. Or am I missing something?

Reply:
The Headers attribute is not used anywhere indeed, but the Cookies one is. See below

https://github.com/nmshd/backbone/blob/admin-ui-security/AdminUi/test/AdminUi.Tests.Integration/API/BaseApi.cs#L45C32-L45C39

Reply:
Okay, then you should use a custom class for the cookies

Reply:
I'm unsure what you mean by custom class. Changing the client does not entail changing the tests, just the BaseApi.

Reply:
> Changing the client does not entail changing the tests, just the BaseApi.

At the moment yes. But as soon as you expose something, it **may** be used wherever you want.

> I'm unsure what you mean by custom class.

Something like this:
```c#
public class Cookie
{
    public string Value { get; init; }
}